README.md

```diff
@@ -54,10 +54,10 @@ Other:
 
 ## **ENVIRONMENT VARIABLES** ##
 Required for simulation:<BR>
-`WORKSPACE`: Defines the absolute path to the directory that contains the Project repository root (called "Caliptra")<BR>
+`CALIPTRA_WORKSPACE`: Defines the absolute path to the directory that contains the Project repository root (called "Caliptra" or "caliptra-rtl")<BR>
+`CALIPTRA_ROOT`: Defines the absolute path to the Project repository root (called "Caliptra" or "caliptra-rtl"). Equivalent to `${CALIPTRA_WORKSPACE}/Caliptra`.<BR>
```

Comment:
CALIPTRA_ROOT should just be defined as $CALIPTRA_WORKSPACE/caliptra-rtl
The rationale for this is:
* we still use the repo name "Caliptra" in our internal repo. so CALIPTRA_ROOT disambiguates this.
* Regression output is placed in a "scratch" folder that is created directly under CALIPTRA_WORKSPACE; the use of a separate variable (instead of just using a relative path $CALIPTRA_ROOT/../scratch) is to reinforce the idea that the repository should be checked out into a dedicated folder to accommodate the scratch folder creation.

Reply:
Thanks for explaining the subtlety.  While you mention reinforcing the idea of checking it into a dedicated folder, would using a .gitignore help in that situation to allow scratch to be written to the git repository?

Reply:
For now the only use-case of CALIPTRA_WORKSPACE is in the verilator regression script to create a scratch, so this solution would work. I can tweak the README to state that CALIPTRA_WORKSPACE is used to indicate the parent address for 'scratch'.

Reply:
lgtm!